packages/core/src/components/DatePicker/YearPicker/YearPicker.module.scss

```diff
@@ -38,6 +38,10 @@
     position: absolute;
     top: 55px;
     pointer-events: all;
+
+    Button {
+      min-width: 66px;
```

Comment:
Isn't there a class? Is it the `button` tag selector?

Reply:
There isn't an existing class on the button (Except for the usual button classes). I considered adding a class but it didn't feel critical just for that. I can add if you feel it is important :)

Reply:
it might cause some css leakage if the specificity is the same in other elements/component, we have a similar issue recently. It's rare but I think it's better to avoid targeting tag selectors all along

Reply:
It is inside the scope of another class, and I made sure that there is no leak. the case we had before was different but Ill change it to be on the safe side :)